### DISCUSSION TITLE
Plucking random() returns complete set

### DISCUSSION THREAD

**Discussion by Norgul** [laravel/framework#52198] [Q&A]
### Laravel Version

11.15.0

### PHP Version

8.2.20

### Description

Executing `SomeModel::all()->random(3)->pluck('id')` will return 3 IDs.
Executing `SomeModel::all()->random(1)->pluck('id')` will return 1 ID

But, executing `SomeModel::all()->random()->pluck('id')` will return all the IDs even though `SomeModel::all()->random()` returns a single model

### Steps To Reproduce

Just execute `SomeModel::all()->random()->pluck('id')`. I would expect to get a single ID.

**Comment by Norgul** [comment]
Actually, same behavior is seen with `SomeModel::first()->pluck('id')`, so this might be by design...even though I do not expect that result to happen

**Comment by crynobone** [comment]
`SomeModel::all()->random(3)` returns a collection instance.

`SomeModel::all()->random()` and `SomeModel::first()` return a Model, and when you attempt to call `pluck`, it simply an alias to `SomeModel::newQuery()->pluck()`
